resolwe_bio/processes/support_processors/seqtk_reverse_complement.py

```diff
@@ -0,0 +1,134 @@
+"""Reverse complement reads with Seqtk."""
+import os
+
+from resolwe.process import Process, DataField, FileField, Cmd, ListField, StringField
+
+
+class ReverseComplementSingle(Process):
+    """Reverse complement FASTQ reads file using Seqtk."""
+
+    slug = 'seqtk-rev-complement-single'
+    process_type = 'data:reads:fastq:single:seqtk:comp'
+    name = 'Reverse complement FASTQ'
+    requirements = {
+        'expression-engine': 'jinja',
+        'executor': {
+            'docker': {
+                'image' : 'resolwebio/common:1.1.0'
+            },
+        },
+        'resources': {
+            'cores': 1,
+            'memory': 16384,
+        },
+    }
+    data_name = '{{ reads|sample_name|default("?") }}'
+    version = '1.0.0'
+
+
+    class Input:
+        """Input fields to process ReverseComplementSingle."""
+
+        reads = DataField('reads:fastq:single', label='Reads')
+
+    class Output:
+        """Output fields."""
+
+        fastq = ListField(FileField(), label='Reverse complemented FASTQ file')
+
+    def run(self, inputs, outputs):
+        """Run the analysis."""
+        name = os.path.basename(inputs.reads.fastq[0].path).strip('.fastq.gz')
+
+        # Concatenate multilane reads
+        (Cmd['cat'][[reads.path for reads in inputs.reads.fastq]] > 'input_reads.fastq.gz')()
+
+        # Reverse complement reads
+        (Cmd['seqtk']['seq', '-r', 'input_reads.fastq.gz'] > f'{name}_complemented.fastq')()
+
+        (Cmd['gzip'][f'{name}_complemented.fastq'])()
+
+        # Save the outputs
+        outputs.fastq = [f'{name}_complemented.fastq.gz']
+
+class ReverseComplementPaired(Process):
+    """Reverse complement FASTQ reads file using Seqtk."""
+
+    slug = 'seqtk-rev-complement-paired'
+    process_type = 'data:reads:fastq:paired:seqtk:comp'
+    name = 'Reverse complement FASTQ (Paired)'
```

Comment:
My suggestion would be to use `(paired-end)` as do other processes on the platform.